Source/Plugins/Core/com.equella.core/js/tsrc/search/SearchPage.tsx

```diff
@@ -115,10 +140,21 @@ const SearchPage = ({ updateTemplate }: TemplateUpdateProps) => {
     }
   }, [searchPageOptions]);
 
+  useEffect(() => {
+    manageSearchPageHistory();
+  }, [filterExpansion]);
```

Comment:
We can make this look after all the cases. Below in `search` where you've called `manageSearchPageHistory` is such that it only saves the options to history on a successful search which is then what drives `pagedSearchResult`. So:

```suggestion
  }, [filterExpansion, pagedSearchResult]);
```